react/features/lobby/components/native/KnockingParticipantList.js

```diff
@@ -5,19 +5,18 @@ import { ScrollView, Text, View, TouchableOpacity } from 'react-native';
 
 import { Avatar } from '../../../base/avatar';
 import { translate } from '../../../base/i18n';
+import { isLocalParticipantModerator } from '../../../base/participants';
 import { connect } from '../../../base/redux';
+import { setKnockingParticipantApproval } from '../../actions';
 import { HIDDEN_EMAILS } from '../../constants';
-import AbstractKnockingParticipantList, {
-    mapStateToProps as abstractMapStateToProps,
-    type Props
-} from '../AbstractKnockingParticipantList';
+import { getKnockingParticipants, getLobbyEnabled } from '../../functions';
 
 import styles from './styles';
 
 /**
  * Component to render a list for the actively knocking participants.
  */
-class KnockingParticipantList extends AbstractKnockingParticipantList {
+class KnockingParticipantList {
```

Comment:
This needs to extend PureComponent and the logic in AbstractKnockingParticipantList needs to be embedded.